setup.py

```diff
@@ -43,6 +43,7 @@
             "h5py >= 2.10.0",
             "smart_open >= 5.0.0",
             "Deprecated >= 1.2.13",
+            "pandas >= 1.0.0",
```

Comment:
remove this dependency. see above

Reply:
You do have this dependency transitively through jams, but I understand if you want to remove direct usage from the package.

Reply:
I think i don't have the authority for this decision 😁

Reply:
so do we add this as a partial dependency? if it comes already with jams, then maybe we just include it here.

Reply:
@magdalenafuentes any thoughts on this?